examples/README.md

```diff
@@ -1,16 +1,26 @@
+## Prerequisites:
+
+Run the following command(s) to ensure proper functionality of the program. 
+
+```shell
+go mod tidy
+```
+
+
 ## Running examples
 
 To run examples execute go run.
 For example:
 
 ```shell
-ACCESS_TOKEN="" go run ./kafkamgmt/kafka_mgmt.go
+ACCESS_TOKEN=`rhoas authtoken` go run ./examples/kafkamgmt/kafka_mgmt.go
 ```
-
+Access token generation uses "rhoas cli" for example purposes. For the real usage you would need to use gocloak library
 ## Setting up urls for instance SDK examples
 
-Instance SDKS like registryinstance and kafkainstance requre extra `API_URL` environment variable that is used 
-to point to the root o the API.
+Instance SDKS like registryinstance and kafkainstance requre extra `API_URL` environment variable that is used to point to the root of the API.
 
 For example 
-API_URL="yourUrl" ACCESS_TOKEN="" go run ./registryinstance/registryinstance.go
\ No newline at end of file
+API_URL="yourUrl" ACCESS_TOKEN=`rhoas authtoken` go run ./examples/registryinstance/registryinstance.go
```

Comment:
MIssing docs how to find that url.

For example for service registry use: rhoas service-registry describe to get `registryUrl` value

Reply:
`rhoas authtoken` should not be used.